templates/firefox-ios.xliff

```diff
@@ -2757,20 +2801,7 @@
       </trans-unit>
     </body>
   </file>
-  <file original="SharedTests/Info.plist" source-language="en" datatype="plaintext">
-    <header>
-      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
-    </header>
-    <body>
-      <trans-unit id="CFBundleName">
-        <source>$(PRODUCT_NAME)</source>
-      </trans-unit>
-      <trans-unit id="CFBundleShortVersionString">
-        <source>16.0</source>
-      </trans-unit>
-    </body>
-  </file>
-  <file original="Storage.strings" source-language="en" datatype="plaintext">
+  <file original="Shared/Supporting Files/Storage.strings" source-language="en" datatype="plaintext">
```

Comment:
File name changed so all existing translations there will be lost (you can look at French for example to see)